README.md

```diff
@@ -34,11 +34,12 @@ Since v1.3, Temporal Web offers optional OAuth SSO authentication. You can enabl
     auth:
       enabled: true # Temporal Web checks this first before reading your provider config
       providers:
-          - label: 'googleoidc'
+          - label: 'google oidc'
```

Comment:
why split this up?

Reply:
i'm thinking to possibly use this label as the label for the sing in button, especially when we have support for many providers. I split because it makes it clearer this label can be any text and doesn't have to follow url-friendly or other formats. Basically to avoid possible question "does this label has to follow some formatting"

Reply:
better to just say that in the comments